utils/frame/try-runtime/cli/src/commands/execute_block.rs

```diff
@@ -19,24 +19,30 @@ use crate::{
 	build_executor, ensure_matching_spec, extract_code, full_extensions, hash_of, local_spec,
 	state_machine_call_with_proof, SharedParams, State, LOG_TARGET,
 };
+use parity_scale_codec::Encode;
 use remote_externalities::rpc_api;
 use sc_service::{Configuration, NativeExecutionDispatch};
 use sp_core::storage::well_known_keys;
 use sp_runtime::traits::{Block as BlockT, Header as HeaderT, NumberFor};
 use std::{fmt::Debug, str::FromStr};
 
 /// Configurations of the [`Command::ExecuteBlock`].
+///
+/// This will always call into `TryRuntime_execute_block`, which can optionally skip state-root
```

Comment:
```suggestion
/// This will always call into `TryRuntime_execute_block`, which can optionally skip the state-root
```

Reply:
Did you mean to write `TryRuntime::execute_block` here?

Reply:
Nope, it is intentional. The method that's called is actually `TryRuntime_execute_block` based on the rules of how runtime APIs are generated.